src/modules/components/proposal/proposalActions/index.ts

```diff
@@ -0,0 +1,10 @@
+import { ProposalActionsAction } from './proposalActionsAction';
+import { ProposalActionsContainer } from './proposalActionsContainer';
+
+export const ProposalActions = {
+    Container: ProposalActionsContainer,
+    Action: ProposalActionsAction,
+};
+
+export * from './actions';
```

Comment:
We usually also export everything defined in the sub-components:
export * from './proposalActionsAction';
export * from './proposalActionsContainer';